pythonx/UltiSnips/snippet/source/file/ultisnips.py

```diff
@@ -105,7 +113,8 @@ def _handle_snippet_or_global(filename, line, lines, python_globals, priority):
     elif snip == 'snippet':
         return 'snippet', (UltiSnipsSnippetDefinition(priority, trig, content,
```

Comment:
make a temporary for the UltiSnipsSnippetDefinition before the return, so that the construction becomes more readable.